src/psyclone/tests/lfric_ref_elem_test.py

```diff
@@ -277,5 +280,166 @@ def test_union_refelem_gen(tmpdir):
             "f4_proxy%data, m3_proxy%data, m4_proxy%data, ndf_w1, undf_w1, "
             "map_w1(:,cell), ndf_w2, undf_w2, map_w2(:,cell), ndf_w3, undf_w3,"
             " map_w3(:,cell), nfaces_re_h, nfaces_re_v, "
-            "out_normals_to_horiz_faces, normals_to_vert_faces, "
+            "normals_to_vert_faces, out_normals_to_horiz_faces, "
```

Comment:
This argument ordering no longer matches the kernel definition.

Reply:
Corrected.